l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -53,6 +57,8 @@ def saveAttachment(self, fatturapa, number):
 
         attach_str = fatturapa.to_xml(self.env)
         attach_vals = {
+            "res_model": "fatturapa.attachment.out",
```

Comment:
Queste modifiche, è meglio includerle in un commit separato (o PR), accompagnato/a da una issue dedicata, così come si era deciso venerdì :wink:  Grazie :pray:

Reply:
hai anche tu ragione

Reply:
#3336
mo' faccio la PR